webpack.config.js

```diff
@@ -1,55 +1,18 @@
 'use strict';
-
-const ROOT = __dirname;
-const path = require('path');
-const webpack = require('webpack');
-
-module.exports = {
-  context: ROOT,
-  entry: {
-    csp: path.join(ROOT, 'src', 'csp'),
-  },
-  output: {
-    path: path.join(ROOT, 'build'),
-    filename: '[name].min.js',
-    libraryTarget: 'umd',
-    library: '[name]',
-  },
-  resolve: {
-    extensions: ['.js'],
-    modules: [
-      path.resolve('./app'),
-      'node_modules',
-    ],
-  },
-  module: {
-    loaders: [
-      {
-        test: /\.js$/,
-        exclude: /node_modules/,
-        loader: 'babel-loader',
-      },
-    ],
-  },
-  devtool: false,
-  plugins: [
-    new webpack.optimize.AggressiveMergingPlugin(),
-    new webpack.optimize.OccurrenceOrderPlugin(),
-    new webpack.DefinePlugin({
-      'process.env.NODE_ENV': "'production'",
-    }),
-    new webpack.LoaderOptionsPlugin({
-      minimize: true,
-      debug: false,
-    }),
-    new webpack.optimize.UglifyJsPlugin({
-      compress: {
-        warnings: false,
-      },
-      output: {
-        comments: false,
-      },
-      sourceMap: false,
-    }),
-  ],
-};
+var es5 = process.env.BABEL_ENV === 'es5'
```

Comment:
Why do we change the style to this one? It is more dynamic, but also is harder to read. Do we have the clear reason why we want this?